tensorflow/python/keras/callbacks_test.py

```diff
@@ -1266,6 +1267,24 @@ def target():
       t.join()
       assert not t.is_alive()
 
+  def test_RemoteMonitor_np_array(self):
+    if requests is None:
+        return
+
+    monitor = keras.callbacks.RemoteMonitor()
+    a = np.arange(1)  # a 1 by 1 array
```

Comment:
Do those two tests cover both np.ndarray and np.generic cases? Is it possible to assert the `send` dict to make sure the items are expected?

Reply:
yes, added assertions